src/nodedef.cpp

```diff
@@ -112,6 +183,64 @@ void NodeBox::deSerialize(std::istream &is)
 		wall_side.MinEdge = readV3F1000(is);
 		wall_side.MaxEdge = readV3F1000(is);
 	}
+	else if (type == NODEBOX_CONNECTED)
```

Comment:
I've rewritten this to be an if statement at the top to determine version number, and then everything else below in a single switch() block, which works well.